src/components/approval-type/ApprovalType.vue

```diff
@@ -1,34 +1,70 @@
 <template>
   <div id="approval-type">
     <v-row no-gutters>
-      <v-col cols="12" sm="3" class="pr-4">
+      <v-col cols="12" sm="3" class="pr-4 pt-5">
         <label id="approval-type-label">Approval Type</label>
       </v-col>
       <v-col cols="12" sm="9">
         <v-radio-group class="payment-group" v-model="approvalTypeSelected" @change="radioButtonChanged">
           <!-- COURT ORDER radio button -->
           <v-radio id="court-order-radio" class="mb-0"
-             :label="getRadioText(ApprovalTypes.VIA_COURT_ORDER)"
-             :value="ApprovalTypes.VIA_COURT_ORDER"
+            :label="getRadioText(ApprovalTypes.VIA_COURT_ORDER)"
+            :value="ApprovalTypes.VIA_COURT_ORDER"
           />
           <v-form ref="courtNumRef" id="court-num-form" v-model="valid" class="mt-4 ml-8">
-            <v-text-field
-              id="court-order-number-input"
-              v-model="courtOrderNumber"
-              label="Court Order Number"
-              :rules="courtOrderNumRules"
-              :disabled="approvalTypeSelected === ApprovalTypes.VIA_REGISTRAR"
-              @input="courtOrderNumberChanged"
-              @update:error="emitValidationError($event)"
-              filled
-            />
+            <v-expand-transition class="pb-0 mb-0">
+              <v-text-field
+                v-if="approvalTypeSelected === ApprovalTypes.VIA_COURT_ORDER"
+                id="court-order-number-input"
+                v-model="courtOrderNumber"
+                label="Court Order Number"
+                :rules="courtOrderNumRules"
+                :disabled="approvalTypeSelected === ApprovalTypes.VIA_REGISTRAR"
+                @input="courtOrderNumberChanged"
+                @update:error="emitValidationError($event)"
+                filled
+              />
+            </v-expand-transition>
           </v-form>
           <!-- REGISTRAR radio button -->
           <v-radio v-if="!isCourtOrderOnly"
-             id="registrar-radio" class="mb-0 pt-2"
-             :label="getRadioText(ApprovalTypes.VIA_REGISTRAR)"
-             :value="ApprovalTypes.VIA_REGISTRAR"
+            id="registrar-radio" class="mb-0 pt-2"
+            :label="getRadioText(ApprovalTypes.VIA_REGISTRAR)"
+            :value="ApprovalTypes.VIA_REGISTRAR"
           />
+          <v-expand-transition>
+            <v-card flat v-if="approvalTypeSelected === ApprovalTypes.VIA_REGISTRAR">
+              <div class="ml-8 mt-3">
+                <span class="v-label">Enter the date the Notice of the Application for Restoration was published in
+                  <br/>
```

Comment:
Don't do this. This sentence only wrapped this way because the uxpin was a narrow viewport. Here, let the sentence wrap automatically depending on the viewport width -- the parent div should take care of setting its size.